mrrc/cli/main.py

```diff
@@ -0,0 +1,113 @@
+"""
+Copyright (c) 2021 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+import argparse
+import locale
+import logging
+
+import pkg_resources
+
+from mrrc import set_logging
+from mrrc.constants import PROG, DESCRIPTION
+from mrrc.util import exception_message
+
+logger = logging.getLogger('mrrc')
+
+
+class CLI(object):
```

Comment:
I'm not quite familiar with python cli programming. Does this mean the argparse is better than the click in python? Is it a recommend way to write python cli program?

Reply:
it's not absolutely better. For some cases click is better but so far from our requirements I don't see a specific need for click. Argparse will give us the controls we need and it'll also let us avoid depending on click since argparse is in-build in python.